proxy/limits/limits.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !windows
+
+// Package limits provides routines to check and enforce certain resource
+// limits on the Cloud SQL client proxy process.
+package limits
+
+import (
+	"fmt"
+	"syscall"
+
+	"github.com/GoogleCloudPlatform/cloudsql-proxy/logging"
+)
+
+var (
+	// For overriding in unittests.
+	getRlimitFunc = syscall.Getrlimit
```

Comment:
Optional nit: usually when faking out for a unit test, I prefer to keep the fully-qualified name in the variable. Something like:

    var (
        syscallGetrlimit = syscall.Getrlimit
        syscallSetrlimit = syscall.Setrlimit
    )

That way, readers can read nearly the same thing in their head as they move through the code.

Reply:
Done.